storage/namespace.go

```diff
@@ -90,6 +102,40 @@ func (n *dbNamespace) Write(ev event.Event) error {
 	return shard.Write(ev)
 }
 
+func (n *dbNamespace) Query(
+	ctx context.Context,
+	q query.ParsedQuery,
+) (query.ResultSet, error) {
+	var res query.ResultSet
+	shards := n.getOwnedShards()
+	for _, shard := range shards {
+		shardRes, err := shard.Query(ctx, q)
```

Comment:
Do we want to parallelize this or create a pool of query threads?

Reply:
For now let's leave it as sequential, can always easily parallelize if needed. Want to measure the what the raw performance is first and can concentrate on perf optimizations afterwards.